levelkdev/swapr-dapp
# Add currency logo next to pooled tokens on User Liquidity

Closes #1339

# Summary

Added token logo next to pooled tokens on User Liquidity to make it easy to read.
- updated UserLiquidity
- updated ValueWithLabel component to accept children
- rename Pool tokens to LP Tokens

<img width="873" alt="image" src="https://user-images.githubusercontent.com/5664434/183075900-f1c5e2b0-77ad-444a-936b-86336d1ba58f.png">

  # To Test

1. Go to liquidity pools
2. Choose a pair
3. Check the logo next to pooled tokens on "Your liquidity" section
